class/defaults.yml

```diff
@@ -133,6 +133,34 @@ parameters:
           helmChartVersion: ${appcat:charts:redis:version}
           imageRegistry: ""
 
+          defaultPlan: standard-1
+          plans:
+            standard-512m:
+              size:
+                cpu: "125m"
+                memory: "512Mi"
+                disk: 16Gi
+            standard-1:
+              size:
+                cpu: "250m"
+                memory: "1Gi"
+                disk: 16Gi
+            standard-2:
+              size:
+                cpu: "500m"
+                memory: "2Gi"
+                disk: 16Gi
+            standard-4:
+              size:
+                cpu: "1"
+                memory: "4Gi"
+                disk: 16Gi
+            standard-8:
+              size:
+                cpu: "2"
+                memory: "8Gi"
+                disk: 16Gi
```

Comment:
We're using the same disk size for all of them as disk resizing actually doesn't work yet..